src/messages/layouts/MessageLayoutContainer.cpp

```diff
@@ -378,13 +378,13 @@ void MessageLayoutContainer::paintSelection(QPainter &painter, int messageIndex,
                 continue;
             }
 
-            for (int i = line.startIndex; i < line.endIndex; i++)
+            for (int ii = line.startIndex; ii < line.endIndex; ii++)
             {
-                int c = this->elements_[i]->getSelectionIndexCount();
```

Comment:
why was this changed at all?

Reply:
The warning was:

`warning C4456: declaration of 'c' hides previous local declaration`